Source/Immutable/Private/Immutable/ImtblBrowserWidget.cpp

```diff
@@ -14,65 +17,68 @@ UImtblBrowserWidget::UImtblBrowserWidget()
 {
 	IMTBL_LOG_FUNCSIG
 
+	JSConnector = NewObject<UImtblJSConnector>(this, "JSConnector");
+	JSConnector->ExecuteJs = UImtblJSConnector::FOnExecuteJsDelegate::CreateUObject(this, &UImtblBrowserWidget::ExecuteJS);
+
 	// WebBrowserWidget->LoadString("<html><head><title>Test</title></head><body><h1>Test</h1></body></html>", TEXT("http://www.google.com"));
 	// InitialURL = TEXT("http://www.google.com");
 	// InitialURL = TEXT("chrome://version");
-    // IPluginManager& PluginManager = IPluginManager::Get();
-    // if (const TSharedPtr<IPlugin> Plugin = PluginManager.FindPlugin("Immutable"))
-    // {
-    //     InitialURL = FString::Printf(TEXT("%s%s"), TEXT("file:///"), *FPaths::ConvertRelativePathToFull(FPaths::Combine(Plugin->GetContentDir(), TEXT("index.html"))));
-    //     IMTBL_LOG("Loading initial url: %s", *InitialURL)
-    // }
-    InitialURL = TEXT("about:blank");
+	// IPluginManager& PluginManager = IPluginManager::Get();
+	// if (const TSharedPtr<IPlugin> Plugin = PluginManager.FindPlugin("Immutable"))
+	// {
+	//     InitialURL = FString::Printf(TEXT("%s%s"), TEXT("file:///"), *FPaths::ConvertRelativePathToFull(FPaths::Combine(Plugin->GetContentDir(), TEXT("index.html"))));
+	//     IMTBL_LOG("Loading initial url: %s", *InitialURL)
+	// }
```

Comment:
Is there a reason why we're keeping this commented code here?

Reply:
That is so that we can quickly test a built index.html file before converting into an assset

Reply:
Might want to add a comment there so others (like myself) know they can use it too.